src/Items/ItemBow.h

```diff
@@ -91,16 +91,15 @@ class cItemBowHandler :
 			}
 			else
 			{
-				Arrow->SetPickupState(cArrowEntity::psNoPickup);
+				ArrowPtr->SetPickupState(cArrowEntity::psNoPickup);
```

Comment:
I checked this and no compiler wouldn't warn of this. There is a tool named `clang-tidy` that has a check named `misc-use-after-move`, but it's in llvm 6..